pybossa/view/account.py

```diff
@@ -386,6 +386,9 @@ def register():
                            email_addr=form.email_addr.data,
                            password=form.password.data,
                            consent=form.consent.data)
+        # guarentee that the full name is well defined wiht no spaces at the beginning or end
+        account['fullname'] = account['fullname'].strip()
```

Comment:
Do you need also handle the heading and trailing spaces for other fields like email_addr and name? If so, it might make sense to strip it when assigning, i.e. form.fullname.data.strip(). Otherwise, it's fine.